src/App.tsx

```diff
@@ -35,3 +37,11 @@ const App = () => {
   )
 }
 export default App
+
+const Nav = styled.nav`
```

Comment:
you only need to use styled when you need actual logic in your sc, otherwise juse use tw:

```suggestion
const Nav = tw.nav`bg-gradient-to-r from-blue-300 to-purple-300 py-16`
```